src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -10,6 +10,44 @@ const itemsPropType = PropTypes.arrayOf(
   })
 );
 
+const renderLink = ({ cssClasses, createURL, refine, templateProps }) => (
+  item,
+  idx,
+  items
+) => {
+  const isLast = idx === items.length - 1;
+  const link = isLast ? (
+    item.name
+  ) : (
+    <a
```

Comment:
Just a thought here: maybe we want a template for the item?

Reply:
Agreed to not change this right now.